wrangler-core/pom.xml

```diff
@@ -68,6 +68,11 @@
       <version>${cdap.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
I don't think you use it in wrangler-core, do you?

Reply:
Was initially running into some circular dependency issues, resolved now